src/zoo.ts

```diff
@@ -0,0 +1,69 @@
+// Інтерфейс Visitor, що описує основні властивості відвідувача
+interface Visitor {
+    name: string;
+    contactInfo: string;
+}
+
+// Інтерфейс Client, що наслідує властивості від Visitor
+interface Client extends Visitor {}
+
+// Клас Ticket для представлення квитка
+class Ticket {
+    constructor(public type: string, public price: number) {}
+}
+
+// Клас Visitor для створення об'єктів відвідувачів
+class Visitor {
+    constructor(public name: string, public contactInfo: string) {}
+}
+
+// Клас Cashier для управління продажем квитків і зберіганням інформації про відвідувачів
+class Cashier {
+    private currentVisitors: Visitor[] = []; // Список поточних відвідувачів
+    private clients: Client[] = []; // Список клієнтів
+    private revenue: number = 0; // Виручка
+
+    // Ціни на квитки залежно від їх типу
+    private ticketPrices: { [key: string]: number } = {
```

Comment:
It seems like a suitable case for using Enums if you want to store constant values.